### ISSUE TITLE
Add Resolves tags to commit messages

### ISSUE THREAD

**Issue by paulmenzel** [xanmod/linux#117]
To link commits fixing issues, [GitHub is able to parse Resolves tags in commit messages](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), and automatically links these commits and issues and even closes them once they land in the main/master branch.

    Resolves: https://github.com/xanmod/linux/issues/XXX

Looking through the git history/log and certain commits would then also have that information, which is nice to have to understand commits better without using GitHub.
